physx/src/scene.rs

```diff
@@ -563,6 +568,11 @@ impl Default for SceneBuilder {
             use_ccd: false,
             enable_ccd_resweep: false,
             solver_type: SolverType::PGS,
+            enhanced_determinism: false,
+            friction_every_iteration: false,
+            stabilization: false,
+            adaptive_force: false,
+            ccd_max_passes: 1,
```

Comment:
Are these the default settings for PhysX?

Reply:
Yes.

Reply:
Perfect!